client/components/AddTestToQueueWithConfirmation/index.jsx

```diff
@@ -149,24 +149,29 @@ function AddTestToQueueWithConfirmation({
                 content={
                     'The report could not be created because an existing ' +
                     'report was found on the reports page with the same AT, ' +
-                    'browser and test plan version. Would you like to return ' +
-                    'the existing report back to the test queue?'
+                    'browser and test plan version. Work is currently ' +
+                    'underway to remove this limitation.'
                 }
                 closeLabel="Cancel"
                 staticBackdrop={true}
-                actions={[
-                    {
-                        label: 'Proceed',
-                        onClick: async () => {
-                            setErrorMessage(false);
-                            if (hasAutomationSupport) {
-                                setShowConfirmation(true);
-                            } else {
-                                await addTestToQueue();
-                            }
-                        }
-                    }
-                ]}
+                // The proceed button is disabled because it will now create
+                // duplicate testPlanReports, and support has not yet been
+                // implemented across the full frontend to account for
+                // duplicates.
```

Comment:
Probably want to defend against this on the backend as well.

Reply:
Sure, I added temporary code to enforce that.